portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormRules.testcase

```diff
@@ -1087,4 +1087,69 @@ definition {
 			locator1 = "FormRules#RULE_BUILDER_FIELD_SELECT_VALUE");
 	}
 
+	@description = "LRQA-47081 - Verify that rules are working for multiple selection field"
```

Comment:
@WYZ-Issac the ticket here should be the LPS, as we're indicating which bug is covered.  Change that to LPS-89260.  The description should also end in the plural "fields" instead of "field".